docs/usage/configuring-plugins-in-stackfiles.rst

```diff
@@ -6,7 +6,7 @@ stackfiles. This section will show you how you can find out which configuration
 is possible for each plugin.
 
 Try the following command to see which plugins are available based on your
-``conantest.py``::
+conanfile profile, such as ``conantest.py``::
 
     $ cloe-launch exec -P ./conantest.py -- usage
```

Comment:
Why not use a command that the user can directly execute, e.g. 
`cloe-launch exec -P ./engine/tests/profile_with_server.py -- usage`

Reply:
Same comment applies to the `cloe-launch exec -P ./conantest.py -- usage basic` further down

Reply:
Because I don't want users to think they have to use our profiles.
I do write "based on *your* contantest.py".

The most I'm willing to do is add a footnote or a note that they can try this out on our profiles.

Reply:
To me it seems rather obvious that any code can be modified by the user according to their needs.

Reply:
Ok, sure. I mean before we moved around the testing scheme, these commands did actually work.